front/src/styles/scss/_uiCoreIntegration.scss

```diff
@@ -68,3 +68,61 @@ img {
     }
   }
 }
+
+// __________________ Select  __________________
```

Comment:
Maybe we can add a todo here to remove this when the bootstrap/tailwind conflict will be resolved ?